packages/edge-login-ui-rn/src/components/themed/EdgeTextField.js

```diff
@@ -0,0 +1,146 @@
+// @flow
```

Comment:
Didn't Andrew create a new text field component specifically for Edge? We should be using his new work instead of react-native-material-textfield.

Reply:
It's not merged yet